coalib/results/TextRange.py

```diff
@@ -89,7 +89,31 @@ def end(self):
         return self._end
 
     def overlaps(self, other):
-        return self.start <= other.end and self.end >= other.start
+        first = False
+        second = False
```

Comment:
first stands for self.start <= other.end
second stands for self.end >= other.start